mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -11,7 +11,7 @@ pipeline {
                     docker {
                         image 'rocm/mlir:miopen_test'
                         args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
-                        label "${XDLOPS == '1' ? 'gfx908 && multi_gpu' : '(gfx908 || gfx900) && multi_gpu'}"
+                        label "${XDLOPS == '1' ? 'gfx908 && multi_gpu' : 'gfx908 && multi_gpu'}"
```

Comment:
Please follow up with Omkar on why test hang on gfx900 nodes. Is it because of too many consumers to them?